src/Command/UpdatePullRequestStatusCommand.php

```diff
@@ -1,18 +1,29 @@
 <?php
 
-namespace AppBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+namespace App\Command;
+
+
+use Github\Client;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Github\Client;
 
-class UpdatePullRequestStatusCommand extends ContainerAwareCommand
+class UpdatePullRequestStatusCommand extends Command
```

Comment:
La commande n'a pas de nom ? où il est généré automatiquement ?

Reply:
Si c'est dans le configure `->setName('deployment:status')`

Reply:
Ah my bad j'avais po vu.

Alors on ne fait plus comme ça depuis Symfony 3.4, voir : 
- https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading
- https://symfony.com/blog/new-in-symfony-3-4-lazy-commands

Reply:
Ah ok, mb je change